evm/src/witness/operation.rs

```diff
@@ -318,7 +318,22 @@ pub(crate) fn generate_get_context<F: Field>(
     state: &mut GenerationState<F>,
     mut row: CpuColumnsView<F>,
 ) -> Result<(), ProgramError> {
-    push_with_write(state, &mut row, state.registers.context.into())?;
+    // Same logic as push_with_write, but we have to use channel 3 for stack constraint reasons.
```

Comment:
Actually, since we are no longer storing the top of the stack in channel 4 (I think that was the reason), can't we use channel 4 both here and for the constraints now?

Reply:
That's true, though it will have to be 3 after https://github.com/0xPolygonZero/plonky2/pull/1291 is merged. We might as well leave it as is.